content/code-security/supply-chain-security/configuration-options-for-dependency-updates.md

```diff
@@ -446,12 +446,12 @@ updates:
 
 #### `rebase-strategy`
 
-By default, {% data variables.product.prodname_dependabot %} automatically rebases open pull requests when it detects conflicts. Use `rebase-strategy` to disable this behavior.
```

Comment:
It's very easy to make some words **bold** and other words *italic* with Markdown. You can even [link to Google!](http://google.com)